client/x_manager_sql_migrations_test.go

```diff
@@ -59,7 +60,7 @@ func TestXXMigrations(t *testing.T) {
 		migratest.MigrationSchemas{Migrations},
 		migratest.MigrationSchemas{createMigrations},
 		CleanTestDB, CleanTestDB,
-		func(t *testing.T, db *sqlx.DB, _, step, steps int) {
+		func(t *testing.T, driver string, db *sqlx.DB, _, step, steps int) {
```

Comment:
Maybe rename this to `dbName`, otherwise this would be considered the driver which would be expected to be `postgres` for CockroachDB.